components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -1666,7 +1666,9 @@ with se_tstep, dt_remap_factor, dt_tracer_factor set to -1
 <use_gw_energy_fix phys="default"> .true.      </use_gw_energy_fix>
 <clubb_C14 phys="default"> 2.5D0      </clubb_C14>
 <clubb_tk1 phys="default"> 268.15D0   </clubb_tk1>
-<dust_emis_fact phys="default"> 1.50D0     </dust_emis_fact>
+
+<!--dust_emis_fact phys="default"> 1.50D0     </dust_emis_fact> YF 04/2022: use for the Zender scheme-->
```

Comment:
Don't comment out the old value.  Remove the line.

Reply:
ok, sounds good. do I need update this file and re-do the push? or this could be taken care during the integration?

Reply:
Yes update the file, commit it, and push again.

Reply:
updated and pushed the file again